frontend/src/lib/lemon-ui/LemonTag/LemonTag.scss

```diff
@@ -33,6 +33,16 @@
         border-radius: 9999px;
     }
 
+    &.LemonTag--breakdown.LemonTag--size-small {
+        padding: 0.375rem 0.5rem;
```

Comment:
Any styling suggestions or improvements are more than welcome; the pill still looks alittle odd imo